### PR TITLE
Change db to postgres and create volunteers table

### PR DESCRIPTION
**Pull request by kylefarmer85**
[ticket](https://trello.com/c/E4L3u19p/166-voma-create-postgresql-table-to-store-volunteer-data)

The app was using sqlite3 as the database, so I changed it to Postgres by adding the 'pg' gem and editing config/database.yml. I created the `volunteers` table based on the description in the ticket.

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -6,7 +6,9 @@ ruby '2.7.1'
 # Bundle edge Rails instead: gem 'rails', github: 'rails/rails', branch: 'main'
 gem 'rails', '~> 6.1.4'
 # Use sqlite3 as the database for Active Record
```
Comment (daleinen7) [inline Gemfile]: Might as well delete this comment too

File: Gemfile [hunk 2]
```diff
@@ -6,7 +6,9 @@ ruby '2.7.1'
...
-gem 'sqlite3', '~> 1.4'
+# gem 'sqlite3', '~> 1.4'
```
Comment (daleinen7) [inline Gemfile]: Might as well delete this comment too

**Review by daleinen7 — CHANGES_REQUESTED** [review]
And not so much a request but would it be worth while to have a file like `example.env` with the environment variables that the project needs (obviously with blank values) so when new people join the team it's clear what their .env needs to get the project up and running?

Or do you think that would that cause more problems than it would solve?

**Comment by kylefarmer85** [comment]
I think that could be useful, maybe on a separate ticket? I'm not sure what environment variables would be needed.

**Comment by Donovon** [comment]
> I think that could be useful, maybe on a separate ticket? I'm not sure what environment variables would be needed.

I agree. We should create a new ticket and discuss it as a group.

**Review by daleinen7 — APPROVED** [review]
Looks good to me.

**Pull request merged**